server/setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup
+
+
+setup(
+    name="kadalu_quotad",
+    version="0.1.0",
+    packages=["kadalu_quotad"],
+    include_package_data=True,
+    install_requires=[""],
+    entry_points={
+        "console_scripts": [
+            "kadalu_quotad = quotad.start:main"
```

Comment:
I am not sure if this line is valid/alright ? @aravindavk ?

I see that if we get "__main__" then we call start, can I directly map the method here?

Reply:
`kadalu_quotad.quotad:crawl` if crawl is the starting function when this CLI is called

Reply:
This can be fixed with the Makefile and Pypi changes.